Assignment 5.Rmd

```diff
@@ -66,17 +91,50 @@ Upload the data "intelligent_tutor_new.csv". This is a data set of a differnt sa
 ```{r}
 #Upload new data
 
-D2 <- 
+D2 <- read.csv("intelligent_tutor_new.csv")
 
 #Generate predicted advice using the predict() command for new students based on tree generated from old students
 
-D2$prediction <- 
+D2$prediction <- predict(grade_ctree,D2)
+
 
 ``` 
 ## Part III
 Compare the predicted advice with the actual advice that these students recieved. What is the difference between the observed and predicted results?
 
 ### To Submit Your Assignment
 
+```{r}
+
+# our obersve
+
+
+#AS THE data shows, 
+# I find the ingore : 58% + 15% + 11% = 84%
+# the monitor shows : 4%
+# the intervenue 7% + 4% = 11%
+
+
+# predicted way we shows:
+
+the_max_data<- apply(D2$prediction,1,which.max)
+
+#we get the no action way:
+
+sum(the_max_data == 3)/length(the_max_data)
+
+# the monitor data we have:
+
+sum(the_max_data == 2)/ length(the_max_data)
+
+#the interventue:
+
+sum(the_max_data == 1)/length(the_max_data)
+
+
+# I actually find the data we I compare, quickly different with the results
+```
```

Comment:
In general, you did a nice job. Maybe you could improve the explanation and interpretation for your coding, which is an important ability for both an analyst and a data scientist. Keep up the good work!